src/components/transactions/transactionDetailView.js

```diff
@@ -154,8 +154,9 @@ class TransactionsDetailView extends React.Component {
           </TransactionDetailViewRow>
 
           <TransactionDetailViewRow shouldShow={
-            this.props.transaction.amount === '0' &&
-            this.props.transaction.recipientId}>
+            (this.props.transaction.amount === '0' &&
+            this.props.transaction.recipientId &&
+            this.props.transaction.type !== 1) }>
```

Comment:
please use available constants for tx's so that is clear which type it is

Reply:
Also, the other two conditions (amount, recipientID) seem to be redundant if you already check it is vote transaction type